src/components/transaction/TransactionTable/index.tsx

```diff
@@ -0,0 +1,295 @@
+import { DateDisplay } from 'components/common/DateDisplay'
+import { RowWithCopyButton } from 'components/common/RowWithCopyButton'
+import { getOrderLimitPrice, formatCalculatedPriceToDisplay, formattedAmount, FormatAmountPrecision } from 'utils'
+import { getShortOrderId } from 'utils/operator'
+import { HelpTooltip } from 'components/Tooltip'
+import StyledUserDetailsTable, {
+  StyledUserDetailsTableProps,
+  EmptyItemWrapper,
+} from '../../common/StyledUserDetailsTable'
+import Icon from 'components/Icon'
+import TradeOrderType from 'components/common/TradeOrderType'
+import { LinkWithPrefixNetwork } from 'components/common/LinkWithPrefixNetwork'
+import { StatusLabel } from 'components/orders/StatusLabel'
+import { media } from 'theme/styles/media'
+import { TextWithTooltip } from 'apps/explorer/components/common/TextWithTooltip'
+import { TokenDisplay } from 'components/common/TokenDisplay'
+import { useNetworkId } from 'state/network'
+import { safeTokenName } from '@gnosis.pm/dex-js'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+const Wrapper = styled(StyledUserDetailsTable)`
+  > thead > tr,
+  > tbody > tr {
+    grid-template-columns: 12rem 7rem repeat(2, minmax(16rem, 1.5fr)) repeat(2, minmax(18rem, 2fr)) 1fr;
+  }
+  tr > td {
+    span.span-inside-tooltip {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      img {
+        padding: 0;
+      }
+    }
+  }
+  ${media.desktopMediumDown} {
+    > thead > tr {
+      display: none;
+    }
+    > tbody > tr {
+      grid-template-columns: none;
+      border: 0.1rem solid ${({ theme }): string => theme.tableRowBorder};
+      box-shadow: 0px 4px 12px ${({ theme }): string => theme.boxShadow};
+      border-radius: 6px;
+      margin-top: 16px;
+      padding: 12px;
+      &:hover {
+        background: none;
+        backdrop-filter: none;
+      }
+    }
+    tr > td {
+      display: flex;
+      flex: 1;
+      width: 100%;
+      justify-content: space-between;
+      margin: 0;
+      margin-bottom: 18px;
+      min-height: 32px;
+      span.span-inside-tooltip {
+        align-items: flex-end;
+        flex-direction: column;
+        img {
+          margin-left: 0;
+        }
+      }
+    }
+    .header-value {
+      flex-wrap: wrap;
+      text-align: end;
+    }
+    .span-copybtn-wrap {
+      display: flex;
+      flex-wrap: nowrap;
+      span {
+        display: flex;
+        align-items: center;
+      }
+      .copy-text {
+        margin-left: 5px;
+      }
+    }
+  }
+  overflow: auto;
+`
+
+const HeaderTitle = styled.span`
+  display: none;
+  ${media.desktopMediumDown} {
+    font-weight: 600;
+    align-items: center;
+    display: flex;
+    margin-right: 3rem;
+    svg {
+      margin-left: 5px;
+    }
+  }
+`
+const HeaderValue = styled.span`
+  ${media.desktopMediumDown} {
+    flex-wrap: wrap;
+    text-align: end;
+  }
+`
+
+function getLimitPrice(transaction: Order, isPriceInverted: boolean): string {
```

Comment:
@maria-vslvn I see tha this is the same function that is in OrdersUserDetailsTable, just with a different variable name form order to transaction. But in the typing I see the transaction variable is still an Order.

I wonder if it makes sense to just reuse the other one?

Reply:
i'll replace `transaction` with `order`, i thought it had a point:)